### PR TITLE
Identify-call-NodeJS-2327

### LINKED ISSUE #2327: [Bug]: Identify Example Uses Wrong Arguments
### Where on segment.com/docs did you see this issue?

https://segment.com/docs/connections/spec/identify/

### What browsers are you seeing the problem on?

_No response_

### Please describe the issue.

Calling identify as shown in the example throws an error. You need to wrap in an object.

**Docs: (wrong)**
```js
analytics.identify("97980cfea0067", {
  name: "Peter Gibbons",
  email: "peter@example.com",
  plan: "premium",
  logins: 5
});
```

**Corrected**
```js
analytics.identify({
  userId: "97980cfea0067",
  traits: {
    name: "Peter Gibbons",
    email: "peter@example.com",
    plan: "premium",
    logins: 5
  }
});
```

**ERROR**
(node:27415) UnhandledPromiseRejectionWarning: AssertionError [ERR_ASSERTION]: You must pass a message object.

**Others having trouble:**
https://github.com/segmentio/analytics-node/issues/192

### PR DESCRIPTION
**Pull request by rchinn-segment**
### Proposed changes
Note based on issue 2327. 

### Related issues (optional)
Closes #2327

### CODE REVIEW

File: src/connections/spec/identify.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ analytics.identify("97980cfea0067", {
   logins: 5
 });
 ```
+> note ""
+> The `identify` call syntax is different for Node.js. Please visit Segment's [Analytics for Node.js](/docs/connections/sources/catalog/libraries/server/node/#identify) page for more information.
```
Comment (stayseesong) [inline src/connections/spec/identify.md:52]:
```suggestion
> The `identify` call syntax is different for Node.js. See the docs for [Analytics for Node.js](/docs/connections/sources/catalog/libraries/server/node/#identify) for more information.
```

Comment (stayseesong) [reply src/connections/spec/identify.md:52]: @rchinn-segment I took a look at the issue that this PR relates to, but is Node.js the only one where the syntax is different?

Comment (rchinn-segment) [reply src/connections/spec/identify.md]: @stayseesong great question. I need to look into it a bit more to see if there are cases other than Node.js where the syntax is different.

File: src/_includes/content/syntax-note.md [hunk 2]
```diff
@@ -0,0 +1 @@
+<div class="premonition info"><div class="fa fa-info-circle"></div> <p markdown=1>Based on the library you use, the syntax in the examples might be different. You can find library-specific documentation from the [Sources Overview](/docs/connections/sources/) page.</p></div>
```
Comment (markzegarelli) [inline src/_includes/content/syntax-note.md:1]:
```suggestion
<div class="premonition info"><div class="fa fa-info-circle"></div> <p markdown=1>Based on the library you use, the syntax in the examples might be different. You can find library-specific documentation on the [Sources Overview](/docs/connections/sources/) page.</p></div>
```
Small nit.

**Review by stayseesong — APPROVED** [review]

**Pull request merged**